### PR TITLE
Add custom config parameter to main.yaml

### PR DESCRIPTION
**Pull request by jeffj254**
The chef_server_ha.yaml template already has code in it to allow custom chef-server.rb config items to be appended to the default chef-server.rb config file.  This parameter was not exposed in main.yaml though and could not be used in a complete stack.

### CODE REVIEW

File: main.yaml [hunk 1]
```diff
@@ -74,6 +74,10 @@ Parameters:
     MaxValue: 16000
   ###############################################################################
   # Chef Server Stack Settings
+  ChefServerCustomConfig:
+    Default: ''
+    Type: String
+    Description: The S3 location of a custom chef-server configuration snippet that will be injected into the chef-server.rb (optional)
```
Comment (irvingpop) [inline main.yaml]: hey @jeffj254 could you please move this to the bottom of this section?   In terms of cognitive burden we want optional things like these to be lower down.

File: main.yaml [hunk 2]
```diff
@@ -249,6 +253,7 @@ Metadata:
       - Label:
           default: "Chef Server Settings"
         Parameters:
+          - ChefServerCustomConfig
```
Comment (irvingpop) [inline main.yaml]: Same request as above, let's stick this at the bottom of this section

**Review by irvingpop — CHANGES_REQUESTED** [review]

File: main.yaml (hunk 1 above)
Comment (jeffj254) [reply main.yaml]: Moved!

File: main.yaml (hunk 2 above)
Comment (jeffj254) [reply main.yaml]: Moved!

**Review by irvingpop — APPROVED** [review]

**Pull request merged**